### PR TITLE
[tests] Use PyTest instead of Nose

### PR DESCRIPTION
**Pull request by abhishek776**
Configure PyTest for Deux and remove all usage of Nose.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@
 dist/
 *.egg-info
 *.egg
+*.eggs/
```
Comment (JamshedVesuna) [inline .gitignore]: You may also need to add a `.cache/` here as well, since pytest creates a local cache.

Comment (abhishek776) [inline .gitignore]: Ah yes will do.

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,9 @@
-[nosetests]
-where = deux/tests
+[aliases]
+test=pytest
+
+[tool:pytest]
```
Comment (JamshedVesuna) [inline setup.cfg]: I also like to add:
```
addopts = --cov=deux --cov-report term --cov-report html:cover --color yes --no-cov-on-fail
```
Which will show a pretty coverage output, so developers can easily see coverage if all tests pass.

File: requirements/test.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
-django-nose==1.4.4
 mock==2.0.0
 coverage>=3.0
+pytest-django==3.0.0
+pytest-runner==2.9
```
Comment (JamshedVesuna) [inline requirements/test.txt]: Moving forward, we should specify a minimum and maximum requirement. This reduces requirement conflicts within a project. Typically, it's good to specify a maximum requirement of the next major version. So here: `pytest-runner>=2.9,<3.0`.

**Pull request merged**